Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1979,6 +2124,49 @@ namespace PlayerCommands
 				return;
 			}
 		}
+		//passed cargo check, now make the distance check
+
+		uint systemId;
+		Vector position;
+		Matrix rotation;
+		pub::Player::GetSystem(client, systemId);
+		pub::SpaceObj::GetLocation(ship, position, rotation);
+		Rotate180(rotation);
+		TranslateX(position, rotation, 1000);

```

Comment:
so this is doing what, pretending the player's position is 1K behind them?

Reply:
1k in front of her, it's the exact copy paste of calculating coordinates for base deployment from PlayerBase() constructor.

Reply:
makes sense